server/src-lib/Data/Time/Clock/Units.hs

```diff
@@ -50,6 +51,9 @@ type Seconds = DiffTime
 seconds :: DiffTime -> DiffTime
 seconds = id
 
+diffTimeToSeconds :: (Integral a) =>  DiffTime -> a
+diffTimeToSeconds = (flip div 1000000000000) . fromInteger . diffTimeToPicoseconds
```

Comment:
Don’t do this! The `Num` instance for `DiffTime` already operates in seconds (which is why the `Seconds` type alias is defined to just be `DiffTime`). Therefore, you can just do this:

```haskell
ghci> let duration :: DiffTime = minutes 2
ghci> round duration :: Integer
120
```